Microsoft.Toolkit.Uwp.UI/Extensions/StatusBarExtensions.cs

```diff
@@ -0,0 +1,158 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.UI;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Provides attached dependency properties for interacting with the <see cref="StatusBar"/> on a window (app view).
+    /// </summary>
+    public static class StatusBarExtensions
+    {
+        /// <summary>
+        /// Gets a value indicating whether StatusBar is supported or not.
+        /// </summary>
+        public static bool IsStatusBarSupported => Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.UI.ViewManagement.StatusBar");
+
+        /// <summary>
+        /// Gets <see cref="Color"/> for <see cref="StatusBar.BackgroundColor"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <returns><see cref="Color"/></returns>
+        public static Color GetBackgroundColor(DependencyObject obj)
```

Comment:
Question that applies to all other properties: Why not attached properties? They could be animated or bound then.